<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_ImageProcessing</class>
 <widget class="QMainWindow" name="ihm_ImageProcessing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>691</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabControles">
     <attribute name="title">
      <string>Controles</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>video devices available</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="in_data1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-999999999999999983222784.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000000000000019884624838656.000000000000000</double>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="start_work">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Commence le job&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>start work</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stop_work">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Emet une requête d'arrêt du job&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>stop work</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="in_data3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-999999999999999983222784.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000000000000019884624838656.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="kill_thread">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Termine puis tue le thread&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>kill thread</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>220</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>work in progress</string>
      </property>
     </widget>
     <widget class="QListWidget" name="video_devices_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="list_algo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>160</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choix de l'algo vidéo à exécuter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <item>
       <property name="text">
        <string>VIDEO_ALGO1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>VIDEO_ALGO2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>VIDEO_ALGO_DUMMY</string>
       </property>
      </item>
     </widget>
     <widget class="QDoubleSpinBox" name="in_data2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-999999999999999983222784.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000000000000019884624838656.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>in_data1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>in_data3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="active_debug">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>190</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Active debug</string>
      </property>
     </widget>
     <widget class="QLed" name="work_status" native="true">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>220</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Binary Led</string>
      </property>
      <property name="whatsThis">
       <string>Led widget</string>
      </property>
      <property name="value" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>in_data2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="init_thread">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Initialise le thread avec la source vidéo sélectionnée&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>init thread</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chck_asBeacon">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>used as beacon</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>(restart of
LaBotBox
is neceassary)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabVideo">
     <attribute name="title">
      <string>Video</string>
     </attribute>
     <widget class="QLabel" name="containerVideo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>240</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1024</width>
        <height>768</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../icons.qrc">:/icons/cancel.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QSlider" name="zoomVideo">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="sliderPosition">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ZOOM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../icons.qrc">:/icons/edit_remove.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>20</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../icons.qrc">:/icons/edit_add.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabBalise">
     <attribute name="title">
      <string>Balise</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>robot_1 angle</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="rob2_angle">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-999999999999999983222784.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000000000000019884624838656.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="fps">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-999999999999999983222784.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000000000000019884624838656.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="rob3_dist">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-999999999999999983222784.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000000000000019884624838656.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>robot_3 dist</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="rob1_dist">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-999999999999999983222784.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000000000000019884624838656.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="rob1_angle">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-999999999999999983222784.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000000000000019884624838656.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>robot_1 dist</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="rob2_dist">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-999999999999999983222784.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000000000000019884624838656.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>robot_3 angle</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="rob3_angle">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>-999999999999999983222784.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000000000000019884624838656.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>robot_2 dist</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>FPS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>robot_2 angle</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>tag aruco -&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>tag aruco -&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>80</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>tag aruco -&gt;</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>48</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>48</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>80</y>
        <width>48</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLed</class>
   <extends>QWidget</extends>
   <header>qled.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
